.github/workflows/CI.yml

```diff
@@ -23,6 +23,15 @@ jobs:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
+        exclude:
+          # Reduce the number of macOS jobs, as fewer can be run in parallel
+          - os: macos-latest
+            julia-version: '1.1'
+          - os: macos-latest
+            julia-version: '1.3'
```

Comment:
```suggestion
```

Reply:
1.3 is where we started using _jll, so I would rather keep it.

Reply:
Sure, whichever way you prefer!